homeassistant/components/plugwise/climate.py

```diff
@@ -129,8 +122,8 @@ def preset_mode(self) -> str | None:
     def extra_state_attributes(self) -> Mapping[str, Any] | None:
         """Return entity specific state attributes."""
         return {
-            "available_schemas": self.device.get("available_schedules"),
-            "selected_schema": self.device.get("selected_schedule"),
+            "available_schedules": self.device.get("available_schedules"),
+            "selected_schedule": self.device.get("selected_schedule"),
```

Comment:
This is a breaking change. Instead, we could also leave them as is and deprecate them (as the select entity will be a full replacement). After two release cycles, the attributes can be fully removed.

Reply:
Didn't think about it that way, but indeed it would replace them.